EpsilonCCS/Inc/Data/AuxBmsData.h

```diff
@@ -23,14 +40,39 @@ enum BatteryPrechargeState
     INVALID_STATE = 5
 };
 
+struct AuxTrip
+{
+    unsigned char chargeTripDueToHighCellVoltage;
+    unsigned char chargeTripDueToHighTemperatureAndCurrent;
+    unsigned char chargeTripDueToPackCurrent;
+    unsigned char dischargeTripDueToLowCellVoltage;
+    unsigned char dischargeTripDueToHighTemperatureAndCurrent;
+    unsigned char dischargeTripDueToPackCurrent;
+    unsigned char protectionTrip;
+};
+
+struct AuxContactor
```

Comment:
Can this be renamed to something like `AuxContactorError`? Just to make it clear that it relates to errors concerning the contactors

Reply:
Or so that it's clear that it's related to debug information

Reply:
I renamed it to `ContactorDebugInfo` to match the protocol.